static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -98,28 +98,33 @@ function BaseDraggableTabList({
         <TabListWrap {...tabListProps} className={className} ref={tabListRef}>
           {persistentTabs.map(item => (
             <Fragment key={item.key}>
-              <Reorder.Item
-                key={item.key}
-                value={item}
-                style={{display: 'flex', flexDirection: 'row'}}
-                as="div"
-                dragConstraints={tabListRef} // Sets the container that the tabs can be dragged within
-                dragElastic={0} // Prevents tabs from being dragged outside of the tab bar
-                dragTransition={{bounceStiffness: 400, bounceDamping: 40}} // Recovers spring behavior thats lost when using dragElastic
-                layout="position"
-              >
-                <Tab
+              <TabItemWrap isSelected={state.selectedKey === item.key}>
+                <Reorder.Item
                   key={item.key}
-                  item={item}
-                  state={state}
-                  orientation={orientation}
-                  overflowing={false}
-                  ref={element => (tabItemsRef.current[item.key] = element)}
-                  variant={tabVariant}
-                />
-              </Reorder.Item>
+                  value={item}
+                  style={{
+                    display: 'flex',
+                    flexDirection: 'row',
+                  }}
+                  as="div"
+                  dragConstraints={tabListRef} // Sets the container that the tabs can be dragged within
+                  dragElastic={0} // Prevents tabs from being dragged outside of the tab bar
+                  dragTransition={{bounceStiffness: 400, bounceDamping: 40}} // Recovers spring behavior thats lost when using dragElastic
+                  layout="position"
+                >
+                  <Tab
+                    key={item.key}
+                    item={item}
+                    state={state}
+                    orientation={orientation}
+                    overflowing={false}
+                    ref={element => (tabItemsRef.current[item.key] = element)}
+                    variant={tabVariant}
+                  />
+                </Reorder.Item>
+              </TabItemWrap>
               <TabDivider
-                layout="position"
+                layout
```

Comment:
Before:

https://github.com/user-attachments/assets/0e4d0a87-ad0e-4d9d-8281-44254f903c1e

After

https://github.com/user-attachments/assets/41434fb6-1fbf-4f90-be54-2f9de2f7c80e

Super nit picky, but I personally like the second one more. The layout=position is less necessary for the tab divider since it's size remains the same.